src/benchmarks/gc/README.md

```diff
@@ -150,7 +150,12 @@ The configuration values that can be used in the test `.yaml` file are described
 
 ## Running
 
-Running the full suite would take a while, so let's just run one:
+The benchmark tests created in the previous step can be run as a whole bundle, or individually.
+This is explained in the following sections.
+
+### Running a Single Test
```

Comment:
I probably wouldn't call this a single *test* since each .yaml could include more than one test. perhaps call it a "scenario"? eg, this is the low memory container scenario and we run multiple tests for this scenario

Reply:
This will require more rephrasing of other parts, especially in this section. Let me draft it and submit it later.